sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyProperties.java

```diff
@@ -266,10 +267,7 @@ void unpackAttributes(Map<String, Object> attributes) {
         this.createdOn = epochToOffsetDateTime(attributes.get("created"));
         this.updatedOn = epochToOffsetDateTime(attributes.get("updated"));
         this.recoveryLevel = (String) attributes.get("recoveryLevel");
-        this.tags = (Map<String, String>) lazyValueSelection(attributes.get("tags"), this.tags);
```

Comment:
Validate that these are not needed in V7.0 SDK.

Reply:
`tags` are only a part of the `KeyBundle` object definition (mapped to the `KeyVaultKey` model) in the Swagger for service version 7.0, they are not included in `KeyAttributes` (mapped to the `KeyProperties` model).